EventMixing/src/Mu2eProductMixer.cc

```diff
@@ -189,6 +189,7 @@ namespace mu2e {
       sr.put(std::move(col), subrunVolInstanceName_);
     }
     if (mixCosmicLivetimes_) {
+      if(generatedEvents_ == 0)throw cet::exception("BADINPUT")<<"Mu2eProductMixer: generated event count =0; was the mixin file opened correctly?" << std::endl;
```

Comment:
Could we have "empty" mixing files where we don't want to throw an exception but just ignore them?